### DISCUSSION TITLE
Could add Atomic CSS in the roadmap?

### DISCUSSION THREAD

**Discussion by MrJmpl3** [vuetifyjs/vuetify#11483] [General] [answered]
### Environment
**Vuetify Version:** 2.x

**Question**

Could add Atomic CSS in the roadmap?

**Other comments**

In this time, Atomic CSS can be a future to optimize the CSS.

Technologies like Atomic CSS-in-JS: https://sebastienlorber.com/atomic-css-in-js/
Or TailwindCSS: https://tailwindcss.com/

A proof of this is the new design of Facebook, theirs reduce the CSS in the Homepage almost 80%. https://engineering.fb.com/web/facebook-redesign/

**Comment by johnleider** [comment, marked as answer]
This is not something we are looking to implement at this time.
